### ISSUE TITLE
How to call line-numbers within ajax load

### ISSUE THREAD

**Issue by etetlow** [PrismJS/prism#1295]
Hi,

Got everything else working dandy now apart from line-numbers which all bunch up at the top when loading with ajax. I got prism and normalize-whitespace working fine by calling them as part of the ajax load but still stuck on line-numbers.

Please advise.

**Comment by Golmote** [comment]
Hi. Please give us an online example reproducing the issue, so that we can investigate.

**Comment by etetlow** [comment]
If you go to http://con.etetlow.co.uk and go to documentation, either the third white dot down or if you click the hamburger you'll see it. When it loads you'll see the line-numbers all bunched up together. If you then refresh the page it'll come directly from that page and the numbers are all fine, so all I need is a way to call line-numbers to reload it on the ajax load.
Thanks.

**Comment by Golmote** [comment]
It seems that you're triggering the highlighting while the elements to be highlighted are not displayed on the page. This is not compatible with the Line numbers plugin, at least with the line wrap support (I think the problem would not occur if there wasn't a `white-space: pre-wrap` set).

The easy solution would be to call `Prism.highlightAll()` manually once the page has been loaded _and displayed_.

**Comment by etetlow** [comment]
I moved the call to `Prism.highlightAll()` until after the element fadein and the result stays the same. I don't think that `Prism.highlightAll()` covers the plugins because normalize whitespace also didn't work until I added 
`Prism.plugins.NormalizeWhitespace.setDefaults({
    'remove-trailing': true,
    'remove-indent': true,
    'left-trim': true,
    'right-trim': true
});`

Is there anything similar to refresh line-numbers because I'm sure that's what's needed. I looked through the plugin and tried a few things but unfortunately without success.
